### PR TITLE
[Databox] SDK refresh to support new API version (2019-09-01)

### PR DESCRIPTION
**Pull request by sashanm**
Azure Spec PR : https://github.com/Azure/azure-rest-api-specs/pull/7317

### CODE REVIEW

**Comment by sashanm** [comment]
@isra-fel  : I closed PR https://github.com/Azure/azure-sdk-for-net/pull/8176 since it was showing others commit as well somehow. so,  I have created a new PR and addressed your code review comments in it. Please take a look and let me know If you have any concerns

Thanks

File: sdk/databox/Microsoft.Azure.Management.DataBox/src/Microsoft.Azure.Management.DataBox.csproj [hunk 1]
```diff
@@ -7,23 +7,20 @@
 		<PackageId>Microsoft.Azure.Management.DataBox</PackageId>
 		<Description>Microsoft Azure Management DataBox library</Description>
 		<AssemblyName>Microsoft.Azure.Management.DataBox</AssemblyName>
-		<Version>1.0.1</Version>
+		<Version>1.2.0</Version>
```
Comment (isra-fel) [inline sdk/databox/Microsoft.Azure.Management.DataBox/src/Microsoft.Azure.Management.DataBox.csproj]: As we discussed, please use 1.1.0

File: sdk/databox/Microsoft.Azure.Management.DataBox/src/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -8,7 +8,7 @@
 [assembly: AssemblyDescription("Provides Microsoft Azure DataBox management functions for managing the Microsoft Azure DataBox service.")]
 
 [assembly: AssemblyVersion("1.0.0")]
-[assembly: AssemblyFileVersion("1.0.1")]
+[assembly: AssemblyFileVersion("1.2.0")]
```
Comment (isra-fel) [inline sdk/databox/Microsoft.Azure.Management.DataBox/src/Properties/AssemblyInfo.cs]: Here too

**Review by isra-fel — CHANGES_REQUESTED** [review]

File: sdk/databox/Microsoft.Azure.Management.DataBox/src/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure DataBox Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure DataBox management functions for managing the Microsoft Azure DataBox service.")]
 
-[assembly: AssemblyVersion("1.0.0")]
-[assembly: AssemblyFileVersion("1.2.0")]
+[assembly: AssemblyVersion("1.1.0")]
```
Comment (isra-fel) [inline sdk/databox/Microsoft.Azure.Management.DataBox/src/Properties/AssemblyInfo.cs]: AssemblyVersion should not be updated unless it's a major version change

**Review by isra-fel — CHANGES_REQUESTED** [review]

**Review by isra-fel — APPROVED** [review]

**Pull request merged**